packages/utils/src/constants.ts

```diff
@@ -1,5 +1,10 @@
+import abi from "./semaphore-abi.json"
+
 // Minimum supported tree depth.
 export const MIN_DEPTH = 1
 
 // Maximum supported tree depth.
 export const MAX_DEPTH = 32
+
+// Semaphore ABI
+export const SemaphoreABI = abi.abi
```

Comment:
Related to the comments above. Here we could export abi and bytecode. Something like:

```ts
export const SemaphoreABI = interface.abi
export const SemaphoreBytecode = interface.bytecode
```

No need to write comments if the variables are self-explanatory.

Reply:
Do you need to export Bytecode separaretly? or do you just want the option of doing so?

Reply:
What do you mean by separately? I think two exports like the ones above should be fine. Wdyt?

Reply:
Hmm no worrries will work on that now